src/API/Helpers/ApiClient.php

```diff
@@ -55,38 +58,15 @@ public function __construct($config)
     {
         $this->basePath      = $config['basePath'];
         $this->domain        = $config['domain'];
-        $this->returnType    = isset( $config['returnType'] ) ? $config['returnType'] : null;
-        $this->headers       = isset($config['headers']) ? $config['headers'] : [];
-        $this->guzzleOptions = isset($config['guzzleOptions']) ? $config['guzzleOptions'] : [];
+        $this->returnType    = $config['returnType'] ?? null;
```

Comment:
?? is null coalescing... so isn't this a no-op?

Reply:
You can't just use `$config['returnType']` or you get an error:

```
❯ php -r '$arr = []; echo $arr["nothing"];'
PHP Notice:  Undefined index: nothing in Command line code on line 1
PHP Stack trace:
PHP   1. {main}() Command line code:0

Notice: Undefined index: nothing in Command line code on line 1

Call Stack:
    0.0002     359800   1. {main}() Command line code:0

❯ php -r '$arr = []; echo $arr["nothing"] ?? "👍";'
👍
```

Reply:
Ah so '??' is null AND nothing coalescing in php?

Reply:
More or less :)